clipper_admin/clipper_admin/deployers/mxnet.py

```diff
@@ -0,0 +1,212 @@
+from __future__ import print_function, with_statement, absolute_import
+import shutil
+import mxnet as mx
+import logging
+import re
+import os
+import json
+
+from ..version import __version__
+from ..clipper_admin import ClipperException
+from .deployer_utils import save_python_function, serialize_object
+
+logger = logging.getLogger(__name__)
+
+MXNET_MODEL_RELATIVE_PATH = "mxnet_model"
+
+
+def create_endpoint(
+        clipper_conn,
+        name,
+        input_type,
+        func,
+        mxnet_model,
+        mxnet_data_shapes,
+        default_output="None",
+        version=1,
+        slo_micros=3000000,
+        labels=None,
+        registry=None,
+        base_image="clipper/mxnet-container:{}".format(__version__),
+        num_replicas=1):
+    """Registers an app and deploys the provided predict function with MXNet model as
+    a Clipper model.
+    Parameters
+    ----------
+    clipper_conn : :py:meth:`clipper_admin.ClipperConnection`
+        A ``ClipperConnection`` object connected to a running Clipper cluster.
+    name : str
+        The name to be assigned to both the registered application and deployed model.
+    input_type : str
+        The input_type to be associated with the registered app and deployed model.
+        One of "integers", "floats", "doubles", "bytes", or "strings".
+    func : function
+        The prediction function. Any state associated with the function will be
+        captured via closure capture and pickled with Cloudpickle.
+    mxnet_model : mxnet model object
+        The MXNet model to save.
+    mxnet_data_shapes : list(int)
```

Comment:
Make this documentation consistent with the `mxnet_data_shapes` parameter documentation for `deploy_mxnet_model`